app/test/features/bluetooth/ui/measurement_multiple_test.dart

```diff
@@ -0,0 +1,85 @@
+
+import 'package:blood_pressure_app/features/bluetooth/logic/characteristics/ble_measurement_data.dart';
+import 'package:blood_pressure_app/features/bluetooth/logic/characteristics/ble_measurement_status.dart';
+import 'package:blood_pressure_app/features/bluetooth/ui/measurement_multiple.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../../util.dart';
+
+
+void main() {
+  testWidgets('should show everything and be interactive', (WidgetTester tester) async {
```

Comment:
<details><summary>I'm still getting a failure:</summary>
<p>

```
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following TestFailure was thrown running a test:
Expected: <1>
  Actual: <0>

When the exception was thrown, this was the stack:
#4      main.<anonymous closure> (file:///home/derdilla/Coding/Flutter/blood_pressure_app/app/test/features/bluetooth/ui/measurement_multiple_test.dart:83:5)
<asynchronous suspension>
#5      testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart:189:15)
<asynchronous suspension>
#6      TestWidgetsFlutterBinding._runTestBody (package:flutter_test/src/binding.dart:1032:5)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)

This was caught by the test expectation on the following line:
  file:///home/derdilla/Coding/Flutter/blood_pressure_app/app/test/features/bluetooth/ui/measurement_multiple_test.dart line 83
The test description was:
  should show everything and be interactive
════════════════════════════════════════════════════════════════════════════════════════════════════
00:22 +131 -3: /home/derdilla/Coding/Flutter/blood_pressure_app/app/test/features/bluetooth/ui/measurement_multiple_test.dart: should show everything and be interactive [E]                                                                     
  Test failed. See exception logs above.
  The test description was: should show everything and be interactive
```

</p>
</details> 

If you don't want to learn the test framework that's fine. Just comment this out and I will do appropriate testing when writing the logic out of the UI.